src/main/java/sirius/db/mixing/properties/LocalDateTimeProperty.java

```diff
@@ -145,4 +145,10 @@ public void describeProperty(JSONObject description) {
     public void contributeToTable(Table table) {
         table.getColumns().add(new TableColumn(this, Types.BIGINT));
     }
+
+    @Override
+    public String getColumnDefaultValue() {
+        // not supported
```

Comment:
```suggestion
        // If a default value is present, this will commonly be a temporal value and thus not a constant
        // (and therefore suitable for a global default value). Therefore we ignore this here. Note that
        // @DefaultValue can still be used to provide a default value to the SQL schema generator).
```